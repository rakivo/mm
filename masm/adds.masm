faddin:
    push 34.5
    push 34.5
    fadd

fsubbin:
    push 530.55
    push 110.55
    fsub

iadding:
    push 113
    push 307
    iadd

isubbin:
    push 213
    push 144
    isub
