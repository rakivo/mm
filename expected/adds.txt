420
69
420
419.99999999999994
69.0
